.pylintrc

```diff
@@ -0,0 +1,62 @@
+[MESSAGES CONTROL]
+disable=fixme,
+    # docstrings aren't required (yet).
+    missing-function-docstring,
+    missing-module-docstring,
+    missing-class-docstring,
+
+    # formatting checks that we're handling with black/isort
+    wrong-import-order,
+    wrong-import-position,
+    ungrouped-imports,
+    line-too-long,
+    superfluous-parens,
+    trailing-whitespace,
+
+    # we'll probably never enable these checks
+    invalid-name,
+    import-error,
+    duplicate-code,
+
+    # disable code-complexity checks for now
+    # TODO: should we configure the thresholds for these rather than just disable?
+    ; too-many-function-args,
```

Comment:
@karlhigley , ha.  I didn't even question it.  I highlighted the lines in vscode and pressed Ctrl+/ to comment them and vscode added the semicolons.  I believe the semi acts as a comment character because I added some of those explicitly to some of the functions.

Reply:
Oh interesting! I thought it might be a comment but then I realized `#` was a comment, and it turns out to be impossible to search for information on this, because all you get is info on the pylint checks about semicolons.